static-website/README.md

```diff
@@ -0,0 +1,12 @@
+# Static Website
+
```

Comment:
this readme is a work in progress. right now it just lists the resources i know i need in order to launch infrastructure for a static site. eventually it will have a detailed description of what everything does and instructions on how to manage it @usydusgduy123 @BellaLong12  fyi

Reply:
nice